README.md

```diff
@@ -10,11 +10,16 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versions
```

Comment:
> We will not split out the patch releases here

This is not semver, which says "A normal version number MUST take the form X.Y.Z where X, Y, and Z are non-negative integers"